community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -40,59 +39,46 @@
     private final PropertyStore propertyStore;
     private final Consumer<StorePropertyCursor> instanceCache;
     private final StorePropertyPayloadCursor payload;
+    private final RecordCursor<PropertyRecord> recordCursor;
 
-    private long nextPropertyRecordId;
     private Lock lock;
+    private boolean initialized;
```

Comment:
This variable has really confusing naming and purpose.
Looks like it should be moved to `StorePropertyPayloadCursor` since it's a state of payload cursor.